opensearch-observability/build.gradle

```diff
@@ -346,27 +351,56 @@ String remoteFileURL = "https://github.com/opensearch-project/observability/rele
-                        @Override
-                        File getAsFile() {
-                            File dir = new File(bwcFilePath + "observability/" + bwcVersion)
-                            if (!dir.exists()) {
-                                dir.mkdirs()
-                            }
-                            File file = new File(dir, "opensearch-observability-1.2.0.0-SNAPSHOT.zip")
-                            if (!file.exists()) {
-                                new URL(remoteFileURL).withInputStream{ ins -> file.withOutputStream{ it << ins }}
+            if (bwcBundleTest) {
+                versions = ["1.3.2", bwcCurrentVersion]
+                nodes.each { node ->
+                    node.extraConfigFile("kirk.pem", file("src/test/resources/security/kirk.pem"))
+                    node.extraConfigFile("kirk-key.pem", file("src/test/resources/security/kirk-key.pem"))
+                    node.extraConfigFile("esnode.pem", file("src/test/resources/security/esnode.pem"))
+                    node.extraConfigFile("esnode-key.pem", file("src/test/resources/security/esnode-key.pem"))
+                    node.extraConfigFile("root-ca.pem", file("src/test/resources/security/root-ca.pem"))
+                    node.setting("plugins.security.disabled", "true")
+                    node.setting("plugins.security.ssl.transport.pemcert_filepath", "esnode.pem")
+                    node.setting("plugins.security.ssl.transport.pemkey_filepath", "esnode-key.pem")
+                    node.setting("plugins.security.ssl.transport.pemtrustedcas_filepath", "root-ca.pem")
+                    node.setting("plugins.security.ssl.transport.enforce_hostname_verification", "false")
+                    node.setting("plugins.security.ssl.http.enabled", "true")
+                    node.setting("plugins.security.ssl.http.pemcert_filepath", "esnode.pem")
+                    node.setting("plugins.security.ssl.http.pemkey_filepath", "esnode-key.pem")
+                    node.setting("plugins.security.ssl.http.pemtrustedcas_filepath", "root-ca.pem")
+                    node.setting("plugins.security.allow_unsafe_democertificates", "true")
+                    node.setting("plugins.security.allow_default_init_securityindex", "true")
+                    node.setting("plugins.security.authcz.admin_dn", "CN=kirk,OU=client,O=client,L=test,C=de")
+                    node.setting("plugins.security.audit.type", "internal_elasticsearch")
+                    node.setting("plugins.security.enable_snapshot_restore_privilege", "true")
+                    node.setting("plugins.security.check_snapshot_restore_write_privileges", "true")
+                    node.setting("plugins.security.restapi.roles_enabled", "[\"all_access\", \"security_rest_api_access\"]")
+                    node.setting("plugins.security.system_indices.enabled", "true")
+                }
+            } else {
+                versions = ["1.2.0",opensearch_version]
```

Comment:
q. how come this works but for bundled test the `-SNAPSHOT` needs to be removed?

Reply:
@joshuali925 For this one, we are getting the min artifacts from URL: `https://artifacts.opensearch.org/releases/core/opensearch/1.2.0/opensearch-min-1.2.0-linux-x64.tar.gz` for released version and `https://artifacts.opensearch.org/snapshots/core/opensearch/2.5.0-SNAPSHOT/opensearch-min-2.5.0-SNAPSHOT-linux-x64-latest.tar.gz` for unreleased snapshot version;
while we are getting the bundle from CI bucket with URL as `https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/2.5.0/latest/linux/x64/tar/dist/opensearch/opensearch-2.5.0-linux-x64.tar.gz` which doesn't contain `-SNAPSHOT` in the path.